dashboard/test/lib/pd/payment/payment_calculator_base_test.rb

```diff
@@ -220,30 +221,28 @@ class PaymentCalculatorBaseTest < ActiveSupport::TestCase
     test 'teacher summaries with deleted teacher account' do
       workshop = create :pd_ended_workshop, num_sessions: 1
 
-      teacher_unqualified = create :pd_workshop_participant,
+      create :pd_workshop_participant,
         workshop: workshop,
         enrolled: true,
         in_section: true,
         attended: true,
         deleted_at: DateTime.now
 
       calculator = PaymentCalculatorBase.instance
-      calculator.expects(:teacher_qualified?).with(teacher_unqualified).returns false
 
       workshop_summary = calculator.calculate(workshop)
       teacher_summaries = workshop_summary.teacher_summaries
       assert teacher_summaries
       assert_equal 1, teacher_summaries.count
 
-      # Unqualified
-      teacher_summaries.find{|p| p.teacher == teacher_unqualified}.tap do |teacher_summary|
-        refute teacher_summary.qualified?
+      teacher_summaries.first.tap do |teacher_summary|
+        assert teacher_summary.qualified?
         assert_equal 1, teacher_summary.days
         assert_equal 6, teacher_summary.hours
-        assert_nil teacher_summary.payment
+        assert_not_nil teacher_summary.payment
       end
```

Comment:
This test was actually incorrect. We do want to report on users who have been deleted, as long as they were active at the time and completed their required puzzles.